config/checker/org.apache.commons.text.astub

```diff
@@ -0,0 +1,8 @@
+import org.checkerframework.checker.tainting.qual.Untainted;
+import org.checkerframework.checker.tainting.qual.Tainted;
+
+package org.apache.commons.text;
+
+class StringEscapeUtils {
+    static @Untainted String escapeHtml4(@Tainted String input);
```

Comment:
A stub for this class has been added in the meantime by Mikołaj, see config/checker/org.apache.commons.astub. Still, it only deals with nullness, so pls merge the two files